api/src/main/scala/de/zalando/play/controllers/ValidationHelpers.scala

```diff
@@ -36,11 +41,23 @@ trait ValidationBase[T] extends Validator {
   // helper to convert failing constraints to errors
   override def errors: Seq[ParsingError] = {
     constraints.map(_(instance)).collect {
-      case Invalid(errors) => errors.toSeq
-    }.flatten.map(ve => ParsingError(ve.messages, ve.args))
+      case Invalid(errors) => errors
+    }.flatten.map(ve => ParsingError(name, ve.messages, ve.args))
   }
 }
 
+trait ValidationTranslator {
+
+  this: I18nSupport =>
```

Comment:
> otherwise it doesn't make sense since we will anyway use the same service together with this one

ah, I see. Well, why not, makes things more clear